### ISSUE TITLE
Let sentences be parsed in smaller parts

### ISSUE THREAD

**Issue by adinaelisheva** [adinaelisheva/apodquiz#5]
Eg: "Despite the 10-degree weather, scientists are excited and researchers are as well" should be able to parse into:
Despite the ___-degree weather, scientists are excited and researchers are as well
Despite the 10-degree weather, _____ are excited and researchers are as well
Despite the 10-degree weather, scientists are _____ and researchers are as well
Despite the 10-degree weather, scientists are excited and _____ are as well

In other words, the comma, 'and', etc should be able to split the sentence into sub-parts for parsing
